nifi-nar-bundles/nifi-enrich-bundle/nifi-enrich-processors/src/main/java/org/apache/nifi/processors/GeoEnrichIPRecord.java

```diff
@@ -102,14 +102,6 @@
             .addValidator(StandardValidators.NON_EMPTY_EL_VALIDATOR)
             .expressionLanguageSupported(ExpressionLanguageScope.FLOWFILE_ATTRIBUTES)
             .build();
-    public static final PropertyDescriptor GEO_ACCURACY = new PropertyDescriptor.Builder()
```

Comment:
Will removing this property cause existing flow deployments to be invalid when upgrading? If the property simply becomes an unused dynamic property, that seems fine, but if it makes the processor invalid, then it seems necessary to keep this property and just update the description.

Reply:
I did a Github search through the Maxmind GeoIp2-Java repo, and it looks like `accuracy` is no longer a field on their response objects. So this wouldn't actually return anything if left in place. It's not a required field, so it shouldn't break anything, but in this case I feel like it might be better to stop the flow until someone looks at it than to let them proceed wondering where the heck the accuracy part went.